frontend/src/components/HealthMetrics.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  faExclamationCircle,
+  faCodeCommit,
+  faCodeFork,
+  faCodePullRequest,
+  faHeart,
+  faStar,
+  faTag,
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React from 'react'
+import type { HealthMetricsProps } from 'types/healthMetrics'
+import GradientRadialChart from 'components/GradientRadialChart'
+import LineChart from 'components/LineChart'
+
+const HealthMetrics: React.FC<{ data: HealthMetricsProps[] }> = ({ data }) => {
+  const openIssuesCountArray = data.map((item) => item.openIssuesCount)
+  const labels = data.map((item, index) => `Day ${index + 1}`)
```

Comment:
We need to update resolvers for all of these data points that we use on charts to also return the data. So this should be not just a number, but a date and number of Open issues at that date. So that it would show the date on the tooltip and on the x axis instead of just Day 1 etc.
Same for other data points.
![Screenshot 2025-06-21 at 12 31 25 PM](https://github.com/user-attachments/assets/bbd5892c-d470-4ca4-a61f-e6d86145f6ce)